src/Neo/SmartContract/Native/NeoToken.cs

```diff
@@ -188,6 +188,9 @@ internal override ContractTask InitializeAsync(ApplicationEngine engine, Hardfor
                 engine.SnapshotCache.Add(CreateStorageKey(Prefix_Committee), new StorageItem(cachedCommittee));
                 engine.SnapshotCache.Add(CreateStorageKey(Prefix_VotersCount), new StorageItem(System.Array.Empty<byte>()));
                 engine.SnapshotCache.Add(CreateStorageKey(Prefix_GasPerBlock).AddBigEndian(0u), new StorageItem(5 * GAS.Factor));
+                // TODO - Add hardfork otherwise storage will be different
+                // After HARD FORK
+                engine.SnapshotCache.Add(CreateStorageKey(Prefix_GasPerBlock).AddBigEndian(0u), new StorageItem(5 * GAS.Factor));
```

Comment:
This also should be kept as is, it's a default and it affects block 0 state changes. And practically this doesn't matter, there is some default, it can be tuned for a particular network. Like people regularly run 1s networks with 5 GASPerBlock networks for tests and it's not an issue.

Reply:
Maybe we need a hard fork for new default, but maybe not needed and can be kept simple as you said.